uk/template_extending/README.md

```diff
@@ -131,4 +131,4 @@
 
 Ось і все! Перевірте, чи ваш сайт все ще коректно працює :)
 
-> Якщо з'явилася помилка `TemplateDoesNotExists`, то це говорить про те, що файлу `blog/base.html` не існує і ви маєте в консолі запущений `runserver`, спробуйте зупинити його (натиснувши Ctrl+C - кнопки Control і C buttons разом) і перезавантажити командою `python manage.py runserver`.
```

Comment:
Hi @das-g It can take some time for translation reviews to happen - rather than waiting for a Ukrainian reviewer to come along and check this. I am also not sure that the word 'buttons' being used originally necessarily means 'keys' is a correct replacement.

If you take the Ukrainian file modification out of this, I'll merge it and we can get your improvements to the English version in place! :sparkles: